lib/mix/tasks/gen.user_from_csv.ex

```diff
@@ -0,0 +1,103 @@
+defmodule Mix.Tasks.Gen.UserFromCsv do
+  use Mix.Task
+  alias Ecto.Multi
+  alias Safira.Repo
+  alias Safira.Auth
+  alias Safira.Accounts.User
+  alias Safira.Accounts.Attendee
+
+  NimbleCSV.define(SeiParser, separator: ",", escape: "\"")
+
+  def run(args) do
+    cond do
+      Enum.empty?(args) ->
+        Mix.shell().info("Needs to receive a file URL.")
+
+      true ->
+        args |> List.first() |> create
+    end
+  end
+
+  defp create(path) do
+    Mix.Task.run("app.start")
+
+    if File.exists?(path) do
+      path
+      |> parse_csv
+      |> create_users
+    else
+      IO.puts("File does not exist")
+    end
```

Comment:
I do not feel very confortable with this condition because what I really want to know is if the file can be opened succesfully, instead of just checking if it exists. However I did not want to use File.open since it actually opens the file, which is not needed because File.stream! will do that a few lines after. Any suggestions.

Reply:
Well, `File.stream!` will raise a `File.Error` exception if it fails to open. So this check is not necessary, but you should Handle the exception instead.